### ISSUE TITLE
WSP Protocol support request

### ISSUE THREAD

**Issue by subhajit-cdot** [ntop/nDPI#2393]
Does nDPI has any plan in the roadmap to support WSP/WTP protocol? 

Reference:
[WSP](https://wiki.wireshark.org/Wireless_Session_Protocol)
[MMS](https://wiki.wireshark.org/Multimedia_Messaging_Service_Encapsulation)

Thanks
Subhajit

**Comment by IvanNardi** [comment]
Not that I'm aware of. BTW, are these protocols still really used?

**Comment by subhajit-cdot** [comment]
Yes, in some legacy 2G/3G devices/scenarios supported by many telecom providers.

**Comment by IvanNardi** [comment]
Example of WSP:  [wap_google.zip](https://github.com/ntop/nDPI/files/14975898/wap_google.zip)

@subhajit-cdot , do you have an example of (interesting) MMS traffic that you can share, please?

**Comment by 0xA50C1A1** [comment]
> Example of WSP: [wap_google.zip](https://github.com/ntop/nDPI/files/14975898/wap_google.zip)
> 
> @subhajit-cdot , do you have an example of (interesting) MMS traffic that you can share, please?

Looks like this ancient artifact is exactly what you're looking for. Yea, it's HTTP, but Content-Type is set as `application/vnd.wap.mms-message`.

[m-send-req-mms.zip](https://github.com/ntop/nDPI/files/14978868/m-send-req-mms.zip)
